src/StoreContext/StoreContext.tsx

```diff
@@ -328,21 +328,25 @@ const createReducer =
 
       case 'updateEditorHeight': {
         const { size } = action.payload;
-        store.setItem('editorHeight', size);
+        const viewportHeight = window.innerHeight;
+        const heightPercentage = (size / viewportHeight) * 100;
```

Comment:
Thoughts on rounding this number?

Wondering whether we should format it as a percentage here too, given there are no calculations on it. Means we dont need to template it everywhere we use it.